login-page.html

```diff
@@ -0,0 +1,65 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=1920px, initial-scale=1.0">
+        <link rel="stylesheet" href="css/login-page.css">
+        <title>LOGIN</title>
+        <link rel="icon" href="images/favicon.ico">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    </head>
+    <body>
+        <div id="content">
+            <img id="ak" src="images/aksu.png" alt="err">
```

Comment:
Check what alt texts are.
If you can use meaningful texts otherwise don't define them at all.